packages/bot-web-ui/src/components/transactions.jsx

```diff
@@ -28,28 +54,29 @@ class Transactions extends React.PureComponent {
                         {localize('Buy price and P/L')}
                     </span>
                 </div>
-                <div className='transactions__content'>
-                    <ThemedScrollbars autoHide style={{ height: 'var(--drawer-scroll-height)' }}>
-                        {elements.map((element, index) => {
-                            switch (element.type) {
-                                case transaction_elements.CONTRACT: {
-                                    const contract = element.data;
-                                    return <Transaction key={`${contract.reference_id}${index}`} contract={contract} />;
-                                }
-                                case transaction_elements.DIVIDER: {
-                                    const run_id = element.data;
-                                    return (
-                                        <div key={run_id} className='transactions__divider'>
-                                            <div className='transactions__divider-line' />
-                                        </div>
-                                    );
-                                }
-                                default: {
-                                    return null;
-                                }
-                            }
-                        })}
-                    </ThemedScrollbars>
+                <div className='' style={{ height: 'calc(var(--drawer-scroll-height) - 7px)' }}>
+                    <AutoSizer>
+                        {({ width, height }) => (
+                            <React.Fragment>
+                                <ThemedScrollbars
+                                    style={{
+                                        height,
+                                        width,
+                                    }}
+                                    onScroll={this.handleScroll}
```

Comment:
Currently, it uses List scrollbar. Can we pass the scroll event from ThemedScrollbars to the List? 
So that it uses ThemedScrollbar instead.

`handleScroll = ({ target }) => {
        const { scrollTop, scrollLeft } = target;
        this.list_ref.Grid.handleScrollEvent({ scrollTop, scrollLeft });
    };`